src/course-unit/course-sequence/sequence-navigation/SequenceNavigationTabs.jsx

```diff
@@ -0,0 +1,62 @@
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { Button } from '@edx/paragon';
+import { Plus as PlusIcon } from '@edx/paragon/icons';
+import { useIntl } from '@edx/frontend-platform/i18n';
+
+import { useIndexOfLastVisibleChild } from '../hooks';
+import messages from '../messages';
+import SequenceNavigationDropdown from './SequenceNavigationDropdown';
+import UnitButton from './UnitButton';
+
+const SequenceNavigationTabs = ({ unitIds, unitId }) => {
+  const intl = useIntl();
+  const [
+    indexOfLastVisibleChild,
+    containerRef,
+    invisibleStyle,
+  ] = useIndexOfLastVisibleChild();
+  const shouldDisplayDropdown = indexOfLastVisibleChild === -1;
+
+  return (
+    <div className="sequence-navigation-tabs-wrapper">
+      <div className="sequence-navigation-tabs-container d-flex" ref={containerRef}>
+        <div
+          className="sequence-navigation-tabs d-flex flex-grow-1"
+          style={shouldDisplayDropdown ? invisibleStyle : null}
+        >
+          {unitIds.map((buttonUnitId) => (
+            <UnitButton
+              key={buttonUnitId}
+              unitId={buttonUnitId}
+              isActive={unitId === buttonUnitId}
+            />
+          ))}
+          {/* TODO: The functionality of the New unit button will be implemented in https://youtrack.raccoongang.com/issue/AXIMST-14 */}
+          <Button
+            className="sequence-navigation-tabs-new-unit-btn disabled"
```

Comment:
If the user does not have the ability to add a new unit, the add unit button should not appear

Reply:
We had an agreement on test coverage for the navigation functionality. Since this functionality is reused with the MFE Learning, we decided not to cover the functionality that comes from the MFE Learning with tests. Tests will be provided for new features (for example, a button to add a new unit) in the following pull requests.